FutureNorthants/northants-website
# Feature/section links image frontend

This PR relies on the following:

- https://github.com/FutureNorthants/northants-design-system/pull/115
- https://github.com/FutureNorthants/northants-website/pull/303

## To Test

- Checkout the design system branch, then run `yalc publish`
- Checkout the backend PR and import config
- Checkout this branch on the frontend and run `rm -rf .next/cache && yalc add northants-design-system && yarn install && yarn dev`
- In a new window, run yarn codegen and git status to ensure there are no changes to the generated graphql.tsx file
- Visit the service landing page that has been updated as per the backend PR instructions and it should now display images for the section that Display Images has been checked
- Service landing pages where the Display Images in the section has not been checked, should display as before
- Visit the service page that has had a meta image added. Inspect the head and it should now have meta tags for og:image and twitter:image.
